docs/docs/noir/concepts/oracles.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Understanding Oracles in Noir - A Comprehensive Guide
```

Comment:
Shorten to match the style of the rest of the pages in this section 
![image](https://github.com/noir-lang/noir/assets/18372439/edab27e0-6811-4b66-924a-40f5afdffcb2)


```suggestion
title: Oracles
```

Reply:
Shit, I always finish by asking ChatGPT to write the frontmatter because of SEO, and forgot to change the title 😂